packages/cross_file/README.md

```diff
@@ -43,3 +43,22 @@ programmatically.
 [![Data on Global support for Blob constructing](https://caniuse.bitsofco.de/image/blobbuilder.png)](https://caniuse.com/blobbuilder)
 
 [![Data on Global support for Blob URLs](https://caniuse.bitsofco.de/image/bloburls.png)](https://caniuse.com/bloburls)
+
+## Testing
+
+This package supports both web and native platforms. Unit tests need to be split
+in two separate suites (because native code cannot use `dart:html`, and web code
+cannot use `dart:io`).
+
+When adding new features, it is likely that tests need to be added for both the
+native and web platforms.
+
+### Native tests
+
+Tests for native platforms are located in the `x_file_io_test.dart`. Tests can
+be run  with `dart test`.
+
+### Web tests
+
+Tests for the web platform live in the `x_file_html_test.dart`. They can be run
+with `dart test -p chrome`.
```

Comment:
So we're only running half of this package's tests in CI?

If so, in the short term we should add a custom test script to this package that runs the others.

Reply:
To clarify: this doesn't need to be done in this PR since it's an existing issue. It's just something we should do ASAP in a follow-up.

Reply:
Yes, this package was weird (web tests didn't use the integration test trick), and now that it doesn't even use flutter, it's even weirder. I can write a custom script, or migrate the web tests to use the examples/integration trick. WDYT @stuartmorgan?

Reply:
Since integration tests are much heavier-weight, and we're already using custom scripts in this repo, it's probably easier to go the custom script route.

Reply:
@stuartmorgan I'll try to add a custom script later tonight.

Reply:
It doesn't have to be *that* ASAP :)

Hopefully it'll be straightforward; ideally it can just be a simple `tool/run_tests.dart` script that just sub-runs `dart test ...`.